Sources_v3/StreamChat/APIClient/Endpoints/CurrentUserEndpoints_Tests.swift

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChat
+import XCTest
+
+final class CurrentUserEndpoints_Tests: XCTestCase {
+    func test_updateCurrentUser_buildsCorrectly() {
+        let userId = UserId.unique
+        let payload: CurrentUserUpdateRequestBody<TestExtraData> = .init(
+            name: .unique, imageURL: .unique(), extraData: TestExtraData(company: .unique)
+        )
+        
+        let users: [String: AnyEncodable] = [
+            "id": AnyEncodable(userId),
+            "set": AnyEncodable(payload)
+        ]
+        let body: [String: AnyEncodable] = [
+            "users": AnyEncodable([users])
+        ]
+        
+        let expectedEndpoint = Endpoint<CurrentUserUpdateResponse<TestExtraData>>(
+            path: "users",
+            method: .patch,
+            queryItems: nil,
+            requiresConnectionId: false,
+            body: body
+        )
+        
+        let endpoint: Endpoint<CurrentUserUpdateResponse> = .updateCurrentUser(id: userId, payload: payload)
+        
+        XCTAssertEqual(AnyEndpoint(expectedEndpoint), AnyEndpoint(endpoint))
+    }
+}
+
```

Comment:
The commit message says `Add update current user endpoint` but I don't see you doing this in the commit :)